sentry-ruby/lib/sentry/configuration.rb

```diff
@@ -184,7 +184,19 @@ def capture_exception_frame_locals=(value)
     # Logger used by Sentry. In Rails, this is the Rails logger, otherwise
     # Sentry provides its own Sentry::Logger.
     # @return [Logger]
-    attr_accessor :logger
+    attr_accessor :sdk_logger
+
+    # @deprecated Use {#sdk_logger=} instead.
+    def logger=(logger)
+      warn "[sentry] `config.logger=` is deprecated. Please use `config.sdk_logger=` instead."
+      self.sdk_logger = logger
+    end
+
+    # @deprecated Use {#sdk_logger} instead.
+    def logger
+      warn "[sentry] `config.logger` is deprecated. Please use `config.sdk_logger` instead."
+      self.sdk_logger = logger
```

Comment:
```suggestion
      self.sdk_logger
```

Reply:
@sl0thentr0py fixed it via 5f0788f83878d18b2f6ba3202977d52b292e61ba + added specs